src/__tests__/statusBar.test.ts

```diff
@@ -1,6 +1,8 @@
 import StatusBar from "../statusBar";
 
-describe("statusBar", () => {
+// TODO (jgzuke) disable this until we migrate away from `vscode`.
+// https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode
+describe.skip("statusBar", () => {
```

Comment:
even though the test is skipped, `yarn test` still fails on the import `> 1 | import { window, StatusBarAlignment } from "vscode";` 🤔 maybe this file could be commented out instead of skipped?